backend/service/k8s/cronjob_test.go

```diff
@@ -0,0 +1,114 @@
+package k8s
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	v1beta1 "k8s.io/api/batch/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8s "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func testCronClientset() k8s.Interface {
+	cron := &v1beta1.CronJob{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:        "testing-cron-name",
+			Namespace:   "testing-namespace",
+			Labels:      map[string]string{"test": "foo"},
+			Annotations: map[string]string{"test": "bar"},
+		},
+	}
+
+	return fake.NewSimpleClientset(cron)
+}
+
+func TestDescribeCron(t *testing.T) {
+	cs := testCronClientset()
+	s := &svc{
```

Comment:
Reduce code duplication by refactoring `testCronClientset` to return the service, also renaming the function to reflect the new behavior.

Reply:
Refactored :)